src/plugins/ncr/components/labels/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import chroma from 'chroma-js';
+import Message from '@gdbots/pbj/Message';
+import labelColors from 'config/labels'; // eslint-disable-line import/no-unresolved
+
+import './styles.scss';
+
+const createStyle = (label) => {
+  if (!labelColors[label]) {
+    return {};
+  }
+
+  const color = chroma(labelColors[label]);
+  return {
+    backgroundColor: color.alpha(0.3).css(),
+  };
+};
+
+const Labels = ({ node }) => node.get('labels', []).map((label) => <span className="label" style={createStyle(label)}>{label}</span>);
```

Comment:
@rsumilang nice concise component. need a unique-valued key prop ( key='myKey' ) on each Label to suppress warning on dashboard page.